external/board/radxa03w/post-image.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -u
+set -e
+set -x
+
+# DTBOs, although we dont use any
+LINUX_DIR=`find $BASE_DIR/build -name 'vmlinux' -type f | xargs dirname`
+mkdir -p $BINARIES_DIR/rockchip/overlays
+if [ -d ${LINUX_DIR}/arch/arm64/boot/dts/rockchip/overlay ]; then
+  cp -a ${LINUX_DIR}/arch/arm64/boot/dts/rockchip/overlay/*.dtbo $BINARIES_DIR/rockchip/overlays
+fi
+
+UBOOT_DIR=`find $BASE_DIR/build -name 'uboot-*' -type d | head -1`
+
+# UBoot Script
+$UBOOT_DIR/tools/mkimage -C none -A arm -T script -d $BR2_EXTERNAL_AA_PROXY_OS_PATH/board/radxa03w/boot.cmd $BINARIES_DIR/boot.scr
```

Comment:
Buildroot's u-boot package can do this for you, and very likely some DTB stuff as well. Might be better to do it the official way.

Reply:
I was trying to help and remove this `mkimage` calls from `post-image` by adding this to defconfig:
```
BR2_TARGET_UBOOT_BOOT_SCRIPT=y
BR2_TARGET_UBOOT_BOOT_SCRIPT_SOURCE="$(BR2_EXTERNAL_AA_PROXY_OS_PATH)/board/radxa03w/boot.cmd"
```
Unfortunately these option has been removed and marked legacy in BR 2020.11 so not sure how it should be called now...
I found similar calls eg here:
https://github.com/buildroot/buildroot/blob/master/board/olimex/a13_olinuxino/post-build.sh
https://github.com/buildroot/buildroot/blob/master/board/boundarydevices/common/post-build.sh#L20
... but I don't know if this is the proper way